packages/client/README.md

```diff
@@ -694,16 +694,17 @@ Proxy publishing and subscribing are handled on the network overlay level. This
 ```js
 
 // Open publish proxy to multiple nodes on stream
-await publishingClient.openProxyConnections(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
+await publishingClient.setProxies(stream, ['0x11111...', '0x22222...'], ProxyDirection.PUBLISH)
```

Comment:
The method doesn't handle a stream, but a stream part. We could highlight that fact in the readme, and  also rename the variable.

(One alternative would be to have the first parameter as `StreamDefinition[]` so that we could set proxy for the whole stream easily.)